### PR TITLE
Upgrade phosphor

### PR DESCRIPTION
**Pull request by conrad-vanl**
## 🐛 Issue

Phosphor is outdated and need new icons in event block pr

## ✏️ Solution

Update phoshpor

Note on the PR size: 

1. Workspaces weren't properly setup, which was messing with dependencies across the packages. The workspace was kinda setup, but anytime yarn did something it spit out a warning aobut workspaces only working on private packages. so I made the base package.json private and it snapped into order
2. There were lint/format issues in some of the files and so the formatting got updated upon save. I can try to revert these changes if too heavy handed

## 🔬 To Test

1.
3.

### CODE REVIEW

**Comment by conrad-vanl** [comment]
Current dependencies on/for this PR:
* **#185** <a href="https://app.graphite.dev/github/pr/ApollosProject/apollos-embeds/185?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ApollosProject/apollos-embeds/185?utm_source=stack-comment).

File: packages/web-shared/package.json [hunk 1]
```diff
@@ -6,6 +6,8 @@
   "private": true,
   "dependencies": {
     "@babel/preset-react": "^7.22.5",
+    "@headlessui/react": "^1.7.18",
+    "polished": "^4.1.2",
```
Comment (conrad-vanl) [inline packages/web-shared/package.json:10]: whoops - this leaked in. I also need it for the eventblock PR (which I'm opening on top of this). I can remove but it's just a bit of extra git steps. IF we're okauy with installing polished on this PR - that would save a few steps for me 😇

Comment (nlewis84) [reply packages/web-shared/package.json:10]: I'm good with keeping it.

**Comment by nlewis84** [comment]
My testing instructions were:

- `yarn` in `web-embeds` and `micro-service`
- Run both of these with `yarn dev`
- Click around the app and see no errors in the terminal

**Review by nlewis84 — APPROVED** [review]
Works for me based on my testing steps 👆 

I'm good with the linter changes. This is the approach we wanted, where if you touch a file, it gets cleaned up.

**Comment by conrad-vanl** [comment]
### Merge activity

* **Mar 1, 1:07 PM EST**: @conrad-vanl merged this pull request with [Graphite](https://app.graphite.dev/github/pr/ApollosProject/apollos-embeds/185).

**Pull request merged**